helm_deploy/keyworker-api/values.yaml

```diff
@@ -47,22 +47,6 @@ generic-service:
       SPRING_DATASOURCE_PASSWORD: "database_password"
       DB_NAME: "database_name"
       DB_ENDPOINT: "rds_instance_endpoint"
-    kw-sqs-instance-output:
```

Comment:
Moved into each environment's config, will revert once we're happy to make same changes in PreProd & Prod